.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "homeassistant/components/frontend/www_static/home-assistant-polymer"]
 	path = homeassistant/components/frontend/www_static/home-assistant-polymer
 	url = https://github.com/balloob/home-assistant-polymer.git
+[submodule "homeassistant/external/verisure"]
```

Comment:
We're moving away from git submodules. Please add your module to a `REQUIREMENTS` variable and `requirements.txt`. See [`sensor.mysensors`](https://github.com/balloob/home-assistant/blob/master/homeassistant/components/sensor/mysensors.py) for an example.

Alternatively, you can also push it to PyPi and add only the package name instead of a reference to GitHub.